apps/pwabuilder/src/script/components/windows-form.ts

```diff
@@ -248,29 +282,77 @@ export class WindowsForm extends AppPackageFormBase {
           <sl-select id="languageDrop" 
             placeholder="Select one or more languages"
             @sl-change=${(e: any) => this.handleLanguage(e)} 
-            .value=${this.packageOptions.resourceLanguage!}
+            value=${this.packageOptions.resourceLanguage!}
```

Comment:
Do we not need the `.` anymore?

Reply:
It used to complain about not having the `.` but after I update shoelace it was complaining about having the `.` The component works as expected without the `.`